desktop_version/src/FileSystemUtils.cpp

```diff
@@ -865,6 +867,33 @@ void FILESYSTEM_enumerateLevelDirFileNames(
     }
 }
 
+std::vector<std::string> FILESYSTEM_enumerateAssets(const char* folder)
+{
```

Comment:
Maybe this could use a comment at the top of the function saying what it does, because it _only_ enumerates level-specific assets and that might be otherwise misleading.